Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ViewExtensions/ViewExtensionsPage.xaml.cs

```diff
@@ -0,0 +1,61 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using Microsoft.Toolkit.Uwp.UI.Extensions;
+using Windows.UI;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Navigation;
+
+namespace Microsoft.Toolkit.Uwp.SampleApp.SamplePages
+{
+    /// <summary>
+    /// An empty page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class ViewExtensionsPage : Page
+    {
+        public ViewExtensionsPage()
+        {
+            InitializeComponent();
+        }
+
+        protected override void OnNavigatedTo(NavigationEventArgs e)
+        {
+            base.OnNavigatedTo(e);
+
+            StatusBarToggle.Visibility = StatusBarExtensions.IsStatusBarSupported ? Visibility.Visible : Visibility.Collapsed;
+        }
+
+        protected override void OnNavigatedFrom(NavigationEventArgs e)
+        {
+            base.OnNavigatedFrom(e);
+
+            StatusBarExtensions.SetIsVisible(this, false);
+
+            ApplicationViewExtensions.SetTitle(this, string.Empty);
+
+            var lightGreyBrush = (Color)Application.Current.Resources["Grey-04"];
+            var greyBrush03 = (Color)Application.Current.Resources["Grey-03"];
+            var greyBrush01 = (Color)Application.Current.Resources["Grey-01"];
+
+            TitleBarExtensions.SetButtonBackgroundColor(this, greyBrush03);
```

Comment:
I was thinking about a kind of recorder: like TitleBarExtensions.SaveState / restoreState